File: codewars/8kyu/OOP Object Oriented Piracy/OOPObjectOrientedPiracy.py
class Ship:
    def __init__(self, draft, crew):
        self.draft = draft
        self.crew = crew
        self.is_worth_it = self.calculate_is_worth_it
        
    def calculate_is_worth_it(self):
       return self.draft - self.crew * 1.5 > 20

File: codewars/8kyu/OOP Object Oriented Piracy/OOPObjectOrientedPiracy.java
public class Ship {
  private final double draft;
  private final int crew;
  private final boolean isWorthIt;
    
  public Ship(double draft, int crew) {
    this.draft = draft;
    this.crew = crew;
    this.isWorthIt = this.draft - this.crew * 1.5 > 20;
  }
  public boolean isWorthIt(){
    return isWorthIt;
  }
 } 